app/src/main/java/edu/uc/groupProject/topten/dto/ListItem.kt

```diff
@@ -8,7 +8,23 @@ package edu.uc.groupProject.topten.dto
  * @param description The description of the listItem
  * @param totalVotes The total number of votes counted towards this listItem. Handled by Firebase.
  */
-data class ListItem(var id:Int, var title: String, var description: String = "", var totalVotes: Int) {
+data class ListItem(
+    var id: Int,
+    var title: String,
+    var description: String = "",
+    var totalVotes: Int
+) {
+
+    init {
+        // Ensure the title is not blank
+        if (title.isBlank()) {
+            throw IllegalArgumentException("Name required")
+        }
+        // Ensure the votes are not negative
+        if (totalVotes < 0) {
+            throw IllegalArgumentException("Votes must be non-negative")
+        }
+    }
```

Comment:
Added an init where I verified the name and vote counts are valid inputs

Reply:
Hmmm.  Good technique.